.rubocop.yml

```diff
@@ -3,7 +3,8 @@ AllCops:
     - "README.md"
     - "Guardfile"
     - "Rakefile"
-    - --force-exclusion "Methods.rb"
+    - methods.rb
+    - ./spec/spec_helper.rb
```

Comment:
- Don't change these files

Reply:
all of the linter error comes from this files.